LayoutTests/platform/mac-sonoma-wk2-lbse-text/svg/custom/gradient-userSpaceOnUse-with-percentage-expected.txt

```diff
@@ -0,0 +1,12 @@
+layer at (0,0) size 800x600
+  RenderView at (0,0) size 800x600
+layer at (0,0) size 300x400
+  RenderSVGRoot {svg} at (0,0) size 300x400
+layer at (0,0) size 300x400
+  RenderSVGViewportContainer at (0,0) size 300x400
+layer at (0,0) size 300x50
+  RenderSVGRect {rect} at (0,0) size 300x50 [fill={[type=SOLID] [color=#000000]}] [x=0.00] [y=0.00] [width=300.00] [height=50.00]
+layer at (0,50) size 300x50
+  RenderSVGRect {rect} at (0,50) size 300x50 [fill={[type=SOLID] [color=#000000]}] [x=0.00] [y=50.00] [width=300.00] [height=50.00]
+layer at (0,100) size 300x300
+  RenderSVGRect {rect} at (0,100) size 300x300 [fill={[type=SOLID] [color=#000000]}] [x=0.00] [y=100.00] [width=300.00] [height=300.00]
```

Comment:
These and others do not refer to gradient pservers, for example the default expectation mentions:
RenderSVGRect {rect} at (0,0) size 300x50 [fill={[type=LINEAR-GRADIENT] [id="gradient1"]}] [x=0.00] [y=0.00] [width=300.00] [height=50.00]

I am not sure we need to dump the pserver itself, but the fill type I guess is needed. WDYT?

Reply:
Indeed, the render tree text dumps  do not know anything about the new resources - follow up pr?

Reply:
Sure, probably more important to land https://github.com/WebKit/WebKit/pull/22352 first anyway.